pdns/recursordist/rec-tcp.cc

```diff
@@ -565,6 +565,9 @@ static void handleRunningTCPQuestion(int fileDesc, FDMultiplexer::funcparam_t& v
           ++iter->second.stats.netmaskMatches;
         }
       }
+      if (t_remotes) {
+        t_remotes->push_back(conn->d_source);
```

Comment:
Am I missing something or are we no longer recording the source over TCP when the proxy protocol is not enabled?

Reply:
Ugh, yes. I'll fix that